extensions/BugModal/template/en/default/bug_modal/edit.html.tmpl

```diff
@@ -278,7 +278,7 @@
       <div>
         <button type="button" id="cancel-btn" class="minor" style="display:none">Cancel</button>
         <button type="button" id="mode-btn" class="major">
-          <span id="mode-btn-readonly">Edit</span>
+          <span id="mode-btn-readonly" title="Toggle editing fields for bug metadata">Edit Fields</span>
```

Comment:
Suggest "Enable editing...", as once you are in Edit mode, there are "Save" and "cancel" buttons, nothing to toggle.